src/sass/app_2/_common/components/settings-dialog.scss

```diff
@@ -1,151 +1,100 @@
 .settings-dialog {
-    position: absolute;
-    width: 320px;
-    right: 1.2em;
-    top: -250px;
-    background: $COLOR_WHITE;
-    height: auto;
-    border-radius: $BORDER_RADIUS;
-    box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.14);
     pointer-events: none;
-    transition: opacity 0.3s ease, transform 0.3s ease;
-    transform: none;
-    opacity: 0;
-    z-index: -1;
 
+    &__language-header {
+        font-size: 14px;
+        font-weight: bold;
+        margin-bottom: 16px;
+        color: $COLOR_LIGHT_BLACK_1;
+    }
+    &__language-container {
+        display: grid;
+        grid-template-columns: repeat(5, 80px);
+        grid-template-rows: repeat(5, 84px);
+        grid-gap: 16px;
+    }
+    &__language {
+        &-link {
+            border-radius: $BORDER_RADIUS;
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+            align-items: center;
+            padding: 16px;
+            color: $COLOR_LIGHT_BLACK_1;
+            text-decoration: none;
+
+            &--active {
+                border: 1px solid $COLOR_LIGHT_GRAY_1;
+                box-shadow: 0 0 0 2px $COLOR_LIGHT_GRAY_2;
+            }
+        }
+        &-flag {
+            width: 36px;
+            height: 24px;
+        }
+        &-name {
+            font-size: 14px;
+            font-weight: normal;
+
+            &--active {
+                font-weight: bold;
+            }
+        }
+    }
     &--enter, &--exit {
-        transform: none;
-        pointer-events: none;
-        z-index: -1;
+        transform: translateY(-50%) scale(0.2, 0.2);
         opacity: 0;
+        pointer-events: none;
     }
     &--enter-done {
-        transform: translateY(-15%) scale(1, 1);
+        transform: translateY(0) scale(1, 1);
         opacity: 1;
         pointer-events: auto;
-        z-index: 1;
     }
     &__container {
         position: relative;
+        width: 736px;
+        background-color: $COLOR_WHITE;
+        border-radius: $BORDER_RADIUS;
+        overflow: hidden;
+        box-shadow: 0 2px 8px 0 $COLOR_LIGHT_BLACK_3;
 
         &--hide {
             display: none;
         }
     }
-    &__header {
-        @include typeface(--small-left-normal-black);
-        @include toEm(padding, 8px 16px, 1.2em);
-        display: block;
-    }
-    & &__tab-header {
-        padding: 1.6em 1.6em 0.8em;
-        font-size: 1.2em;
-    }
-    &__tab-content {
-        display: flex;
-        justify-content: center;
-        align-items: flex-start;
-        height: 216px;
-        background: $COLOR_LIGHT_BLACK_4_SOLID;
+}
+/**
+ * @define dialog-header
+ */
+.dialog-header {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    box-shadow: 0 2px 8px 0 var($COLOR_LIGHT_BLACK_3);
+
+    &__sidebar {
+        width: 200px;
+        font-size: 16px;
+        line-height: 24px;
+        padding: 16px;
+        font-weight: bold;
+        color: $COLOR_LIGHT_BLACK_1;
```

Comment:
could you use the `@typeface` mixin here, please? 
line 80, 81, 83 & 84 can be simplified to `@include typeface(--title-left-bold-black);`

Reply:
Absolutely.

Reply:
was this done?

Reply:
No. I'm doing it now.